spec/system/add_activity_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+describe 'adding a new activity' do
+
+  let(:school_name) { 'Active school'}
+  let(:activity_type_name) { 'Exciting activity' }
+  let(:other_activity_type_name) { 'Exciting activity (please specify)' }
+  let(:activity_description) { 'What we did' }
+  let(:custom_title) { 'Custom title' }
+  let!(:school) { create_active_school(name: school_name)}
+  let!(:admin)  { create(:user, role: 'school_user', school: school)}
+  let!(:activity_type) { create(:activity_type, name: activity_type_name, description: "It's An #{activity_type_name}") }
+  let!(:other_activity_type) { create(:activity_type, name: other_activity_type_name, description: nil, custom: true) }
+
+  before(:each) do
+    sign_in(admin)
+    visit school_path(school)
+    click_on('Choose an activity')
+    click_on('Record your activity')
+  end
+
+  it 'allows an activity to be created without title', js: true do
```

Comment:
Revisit this - have one JS test which tests description and custom stuff, one without JS which can test everything else